.github/CODEOWNERS

```diff
@@ -1,3 +1,5 @@
+*                                         @primer-io/acceptance-mobile-ios
```

Comment:
@psedge I am intending to use [this team](https://github.com/orgs/primer-io/teams/acceptance-mobile-ios) here - does it need to be exposed to our repos?

While I have you, since team checkout is no more, should we clone `checkout-pci-reviewers` to a new name (`acceptance-sdk-pci-reviewers`?) to follow the same pattern. We can remove `checkout-pci-reviewers` once everything is changed over?

Reply:
Yes, it has to be a direct member.